pallets/capacity/src/benchmarking.rs

```diff
@@ -125,7 +139,47 @@ benchmarks! {
 		assert_last_event::<T>(Event::<T>::EpochLengthUpdated {blocks: epoch_length}.into());
 	}
 
+	change_staking_target {
+		let caller: T::AccountId = create_funded_account::<T>("account", SEED, 5u32);
+		let from_msa = 33;
+		let to_msa = 34;
+		// amount in addition to minimum
+		let from_msa_amount = 32u32;
+		let to_msa_amount = 1u32;
+
+		register_provider::<T>(from_msa, "frommsa");
+		register_provider::<T>(to_msa, "tomsa");
+		setup_provider_stake::<T>(&caller, &from_msa, from_msa_amount.into());
+		setup_provider_stake::<T>(&caller, &to_msa, to_msa_amount.into());
+		let restake_amount = 11u32;
+
+	}: _ (RawOrigin::Signed(caller.clone(), ), from_msa, to_msa, restake_amount.into())
+	verify {
+		assert_last_event::<T>(Event::<T>::StakingTargetChanged {
+			account: caller,
+			from_msa,
+			to_msa,
+			amount: restake_amount.into()
+		}.into());
+	}
+
+	provider_boost {
```

Comment:
Question: I can see that we have weights for `provider_boost` without the benchmarks since you just added it in this PR. Where does those weights come from?

Reply:
I can't remember but since this is a months-old branch that has been rebased against the previous which was again a months-old branch that had to be rebased against main, I think I must have run benchmarks locally to get those weights, and it got clobbered in the rebase.

Reply:
Also fixed these issues.  See newly generated weights, and change_staking_target now uses its correct weight function.